src/NServiceBus.RabbitMQ/Configuration/ConnectionStringParser.cs

```diff
@@ -66,7 +66,7 @@ public ConnectionConfiguration Parse(string connectionString)
 
             if (ContainsKey("prefetchcount"))
             {
-                var message = "The 'PrefetchCount' connection string option has been removed. Use 'EndpointConfiguration.LimitMessageProcessingConcurrencyTo' and 'EndpointConfiguration.PrefetchCountPerMessageProcessor' instead.";
+                var message = "The 'PrefetchCount' connection string option has been removed. Use 'TransportExtensions<RabbitMQTransport>.PrefetchCount' instead.";
```

Comment:
I'm not sure what I think about having to refer to the class we use for the extension methods here, but it seemed less awkward that trying to say something about EndpointConfiguration.UseTransport<RabbitMQTransport>.PrefetchCount.

What do you think, should we go with that instead? More verbose, but possibly clearer.